test/Microsoft.AspNetCore.Razor.Runtime.Test/Runtime/TagHelpers/TagHelperDesignTimeDescriptorFactoryTest.cs

```diff
@@ -368,50 +337,35 @@ public static TheoryData CreateAttributeDescriptor_PropertyDocumentationData
                         }
                     },
                     {
-                        CreateDocumentationTagHelperType(defaultLocation, defaultCodeBase),
+                        typeof(DocumentedTagHelper),
                         nameof(DocumentedTagHelper.SummaryProperty),
+                        defaultLocation,
                         new TagHelperAttributeDesignTimeDescriptor
                         {
                             Summary = PropertySummary
                         }
                     },
                     {
-                        CreateDocumentationTagHelperType(defaultLocation, defaultCodeBase),
+                        typeof(DocumentedTagHelper),
                         nameof(DocumentedTagHelper.RemarksProperty),
+                        defaultLocation,
                         new TagHelperAttributeDesignTimeDescriptor
                         {
                             Remarks = PropertyRemarks
                         }
                     },
                     {
-                        CreateDocumentationTagHelperType(defaultLocation, defaultCodeBase),
-                        nameof(DocumentedTagHelper.RemarksAndSummaryProperty),
-                        new TagHelperAttributeDesignTimeDescriptor
-                        {
-                            Summary = PropertyWithSummaryAndRemarks_Summary,
-                            Remarks = PropertyWithSummaryAndRemarks_Remarks
-                        }
-                    },
-                    {
-                        CreateDocumentationTagHelperType(nonExistentLocation, codeBase: null),
+                        typeof(DocumentedTagHelper),
                         nameof(DocumentedTagHelper.RemarksAndSummaryProperty),
+                        nonExistentLocation,
                         null
                     },
                     {
-                        CreateDocumentationTagHelperType(location: null, codeBase: nonExistentCodeBase),
+                        typeof(DocumentedTagHelper),
                         nameof(DocumentedTagHelper.RemarksAndSummaryProperty),
+                        invalidLocation,
                         null
                     },
-                    {
-                        CreateDocumentationTagHelperType(invalidLocation, codeBase: null),
-                        nameof(DocumentedTagHelper.RemarksAndSummaryProperty),
-                        null
-                    },
-                    {
-                        CreateDocumentationTagHelperType(location: null, codeBase: invalidCodeBase),
-                        nameof(DocumentedTagHelper.RemarksAndSummaryProperty),
-                        null
-                    }
```

Comment:
Guessing at the reason for the smaller data set above: Is this also about eliminating tests of invalid `CodeBase` values?